server/models/handlers/workerTimeslot.js

```diff
@@ -10,6 +10,18 @@ async function addWorkerTimeslot(slotId, schoolId, workerId, status, date) {
         return false;
     }
 };
+
+async function updatingTheWorkerAvailability(status){
+    try {
+        await timeslotModel.insertWorkerTimeslot(status);
+        return true;
+    } catch (error) {
+        console.log('Error occurred in updatingTheWorkerAvailability method: ', error);
+        return false;
+    }
+};
+
 module.exports = {
-    addWorkerTimeslot
-}
+    addWorkerTimeslot,
+    updatingTheWorkerAvailability,
```

Comment:
Hi Simran,

I'm curious: What happens if the `status` is 'null'?

Reply:
Hi Nancy, 

That's a great question! I will communicate with the team regarding that question. I was assuming that the status can only be, "available" and "unavailable"? You've addressed a good point and I will research on this further! 

Thanks! 

Simran